Sources/Node/CommandsDeserializer.cpp

```diff
@@ -6,16 +6,20 @@
 #include "Commands/AddProcessCommand.h"
 
 
-Command CommandsDeserializer::parseToCommand(std::string json) {
+std::shared_ptr<Command> CommandsDeserializer::parseToCommand(std::string json) {
 
   std::string typeSpecificator = "messageType";
+  Json::Reader reader;
   Json::Value root;
+  bool parsingSucceed = reader.parse(json.c_str(), root);
```

Comment:
parsingSucceed doesn't really make much sense in English in this context; parsingSuccess or parsingSucceeded would